### PR TITLE
fix #31

### LINKED ISSUE #31: Dealing with save size explosions from overpopulated characters?
I wrote a kero adder last night to add one character per second:
https://github.com/steve02081504/kero-adder
Watching discord chat this morning I saw this:
```
💜 Zi  //watch out for scams — 今天04:25
good, good 😂 you have avoided the nightmare i once created for myself
don't uh. don't make characters with high numbers appear in YAYA, because YAYA will have to generate an array entry for them... in two arrays... which can bloat your save file a ton! because character 9000 has to be placed at array index 9000, which means 0-8999 have to be generated as empty elements if they don't exist, and... then they never go away :crylaugh:
```
I think this might be a fixable issue and easily triggered by jsstp.

### PR DESCRIPTION
**Pull request by nikolat**

### CODE REVIEW

File: yaya_base/shiori3.dic
Comment (nikolat) [inline yaya_base/shiori3.dic:1]: > //以下は保存
> //SHIORI3FW.LastSurface
> //SHIORI3FW.IsVisible

これ何か理由あるんでした？ @ponapalt

**Review by nikolat — COMMENTED** [review]
commitしてから何か気付いてしまった

File: yaya_base/shiori3.dic
Comment (steve02081504) [reply yaya_base/shiori3.dic:1]: I'm guessing it's possible to make ghost aware of the last used Surface and front and back even on the next boot or something?

Comment (nikolat) [reply yaya_base/shiori3.dic:1]: There are ghosts who respond to `OnSurfaceRestore` differently depending on the last surface.
However, it seems unnatural to remember the surface from the last boot. 🤔

**Comment by nikolat** [comment]
https://ukadon.shillest.net/@ponapalt/111625878095010255

**Pull request merged**